acmw-db-app/styles/Reset.module.css

```diff
@@ -0,0 +1,205 @@
+/* COLOR SCHEME
+  light purpur: #BA9ADF
+  bright purpur: #9A49CE
+  white: #FFFFFF
+  dark gray: #373738
+*/
+
```

Comment:
I'm wondering if we should put all this in `globals.css`

Reply:
I think your whole style css should go into global because I used most of it myself. It has good coloring and is a good base.